src/main/java/org/opensearch/ad/transport/DeleteModelTransportAction.java

```diff
@@ -65,7 +66,8 @@ public DeleteModelTransportAction(
         ModelManager modelManager,
         FeatureManager featureManager,
         CacheProvider cache,
-        ADTaskCacheManager adTaskCacheManager
+        ADTaskCacheManager adTaskCacheManager,
+        NodeStateManager stateManager
```

Comment:
We have had a NodeStateManager. The name transportStateManager is outdated though.